apps/spruce/src/components/Settings/Form.test.tsx

```diff
@@ -56,19 +56,25 @@ describe("context-based form", () => {
     render(<Component tab="bar" validate={barValidator} />, {
       wrapper: TestProvider,
     });
-    await user.clear(screen.getByLabelText("Age"));
-    expect(screen.getByLabelText("Age")).toHaveValue("");
+    await user.clear(screen.getAllByLabelText("Age")[0]);
```

Comment:
The reason this broke is due to this change https://github.com/mongodb/leafygreen-ui/blob/main/packages/text-input/CHANGELOG.md#aria-label 

The underlying text input component needs to be updated to follow the new aria label rules